lms/djangoapps/courseware/views/views.py

```diff
@@ -1637,6 +1637,7 @@ def get_financial_aid_courses(user):
     for enrollment in CourseEnrollment.enrollments_for_user(user).order_by('-created'):
 
         if enrollment.mode != CourseMode.VERIFIED and \
+                enrollment.course_overview and \
```

Comment:
@schenedx @ahsan-ul-haq have either of you actually looked into why some course runs fail to load from the modulestore? I'm guessing that's why there were problems here. If a course run is missing a course overview, one is lazily created unless there's a problem loading the course from the modulestore, in which case an IOError is [raised](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/content/course_overviews/models.py#L232-L236).